lib/cartopy/crs.py

```diff
@@ -1496,7 +1528,7 @@ def __init__(self, central_longitude=0.0, central_latitude=0.0,
             if (5, 0, 0) <= PROJ4_VERSION < (5, 1, 0):
                 warnings.warn(
                     'The Orthographic projection in Proj between 5.0.0 and '
-                    '5.1.0 incorrectly transforms points. Use this projection '
+                    '5.0.1 incorrectly transforms points. Use this projection '
```

Comment:
This message doesn't agree with the version check above.

Reply:
I've changed the message to be closer to what the Robinson one is, so it should be clearer.

Reply:
I've changed the message to be closer to what the Robinson one is, so it should be clearer.